packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -85,54 +133,92 @@ const ApplicationDetails = () => {
+                          onRemove={() => {
+                            removeRedirectUri(index);
+                          }}
+                          {...register(`oidcClientMetadata.redirectUris.${index}.uri`)}
                         />
-                      </FormField>
-                      <FormField title="admin_console.application_details.user_info_endpoint">
-                        <CopyToClipboard
-                          className={styles.copy}
-                          value={oidcConfig.userinfo_endpoint}
+                      ))}
+                      <AddAnotherButton
+                        onClick={() => {
+                          appendRedirectUri({ uri: '' });
+                        }}
+                      />
+                    </FormField>
+                    <FormField
+                      title="admin_console.application_details.post_sign_out_redirect_uri"
+                      className={styles.listFields}
+                    >
+                      {postLogoutRedirectUriFields.map((field, index, fields) => (
+                        <InputWithDelete
+                          key={field.id}
+                          isDeletable={index > 0 || fields.length > 1}
+                          onRemove={() => {
+                            removePostLogoutRedirectUri(index);
+                          }}
+                          {...register(`oidcClientMetadata.postLogoutRedirectUris.${index}.uri`)}
                         />
-                      </FormField>
-                    </>
-                  )}
-                </div>
-                <div className={styles.submit}>
-                  <Button
-                    htmlType="submit"
-                    type="primary"
-                    title="admin_console.application_details.save_changes"
-                  />
-                </div>
-              </form>
-            </div>
+                      ))}
+                      <AddAnotherButton
+                        onClick={() => {
+                          appendPostLogoutRedirectUri({ uri: '' });
+                        }}
+                      />
+                    </FormField>
+                  </>
+                )}
+                {isAdvancedSettings && (
+                  <>
+                    <FormField title="admin_console.application_details.token_endpoint">
+                      <CopyToClipboard
+                        className={styles.textField}
+                        value={oidcConfig.token_endpoint}
+                      />
+                    </FormField>
+                    <FormField title="admin_console.application_details.user_info_endpoint">
+                      <CopyToClipboard
+                        className={styles.textField}
+                        value={oidcConfig.userinfo_endpoint}
+                      />
+                    </FormField>
+                  </>
+                )}
```

Comment:
nit: since the react dom itself could grow huge. Instead of putting them all together, prefer to extract some of these sections into a useMemo variable.  
so in your main body simply { isAdvancedSettings ? advanceSection : defaultSection }